ratelimit/redis.go

```diff
@@ -121,6 +124,7 @@ func (c *clusterLimitRedis) Allow(ctx context.Context, clearText string) bool {
 	if failed {
 		allow = !c.failClosed
 		setError(span)
+		c.logError("Failed to redis to determine if operation is allowed", err)
```

Comment:
Please rephrase into simple sentence and add %v format specifier: `Failed to determine if operation is allowed: %v` (also elsewhere)